aspnetcore/log-mon/metrics/metrics/samples/metric-tests/BasicTests.cs

```diff
@@ -17,7 +17,7 @@ public async Task Get_RequestCounterIncreased()
         var client = _factory.CreateClient();
         var meterFactory = _factory.Services.GetRequiredService<IMeterFactory>();
         var collector = new MetricCollector<double>(meterFactory,
-            "Microsoft.AspNetCore.Hosting", "http.server.request.duration");
```

Comment:
These values are from RC1. There is no much point changing them as RC1 comes out soon and they'll change back again.